synapse/config/registration.py

```diff
@@ -15,19 +15,43 @@
 
 from ._base import Config
 
+from synapse.util.stringutils import random_string_with_symbols
+
+import distutils.util
+
 
 class RegistrationConfig(Config):
 
     def __init__(self, args):
         super(RegistrationConfig, self).__init__(args)
-        self.disable_registration = args.disable_registration
+
+        self.disable_registration = bool(
+            distutils.util.strtobool(str(args.disable_registration))
```

Comment:
This seems like a NOOP to me. Isn't ``args.disable_registration`` a ``bool`` to begin with? In which case this is just doing ``bool -> str -> bool``. In what cases will ``args.disable_registration`` not be a ``bool``?